test/client/scraper/test_dropbox_metadata.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+import json
+from client.query_utils import QueryParams
+from client.constants import ResponseType
+import client.api_handler
+import client.api
+
+
+class TestDropboxMetadata(unittest.TestCase):
+
+    test_url = "https://www.dropbox.com/s/t8hep77ze6jypj6/Screenshot%20from%202014-01-07%2020%3A33%3A13.png?dl=0"
```

Comment:
check for 404? (file not found)
also check for invalid dropbox url (https://www.dropbox.com/s/invalid)